deepomatic/api/http_helper.py

```diff
@@ -22,25 +22,36 @@
 THE SOFTWARE.
 """
 
-import os
+import functools
 import json
-import requests
-import sys
+import os
 import platform
-from requests.structures import CaseInsensitiveDict
-from six import string_types
+import sys
 
-from deepomatic.api.exceptions import DeepomaticException, BadStatus
+import requests
+from deepomatic.api.exceptions import BadStatus, DeepomaticException
+from deepomatic.api.http_retry import HTTPRetry
 from deepomatic.api.version import __title__, __version__
+from requests.structures import CaseInsensitiveDict
+from six import string_types
 
 API_HOST = 'https://api.deepomatic.com'
 API_VERSION = 0.7
 
 ###############################################################################
 
 
+class RequestsTimeout(object):
+    FAST = (3.05, 10.)
+    MEDIUM = (3.05, 60.)
+    SLOW = (3.05, 600.)
+
+
 class HTTPHelper(object):
-    def __init__(self, app_id=None, api_key=None, verify_ssl=None, host=None, version=API_VERSION, check_query_parameters=True, user_agent_prefix='', user_agent_suffix='', pool_maxsize=20):
+    def __init__(self, app_id=None, api_key=None, verify_ssl=None,
+                 host=None, version=API_VERSION, check_query_parameters=True,
+                 user_agent_prefix='', user_agent_suffix='', pool_maxsize=20,
+                 requests_timeout=RequestsTimeout.FAST, http_retry=None):
```

Comment:
```suggestion
                 requests_timeout=RequestsTimeout.FAST, http_retry=HTTPRetry.DEFAULT):
```
(and update `Client` doc?)

Reply:
I prefer not doing so because if `self.http_retry` is modified internally it will modify the `DEFAULT`.

Reply:
it was the same issue with the previous version where in the `__init__`  we had `self.http_retry = http_retry or HTTPRetry.DEFAULT`.

In any case your point is valid; and I don't know how to easily freeze an object.

Maybe just document we use `HTTPRetry()` by default in `Client` doc?

Reply:
As discussed, we will default http_retry=HTTPRetry() and warn the developer to not modify self.http_retry.